### PR TITLE
Add Debug log of torch_model_blacklist.txt

### PR DESCRIPTION
**Pull request by AmosLewis**

### CODE REVIEW

File: generate_sharktank.py [hunk 1]
```diff
@@ -194,29 +194,30 @@ def is_valid_file(arg):
         help="""Contains the file with torch_model name and args.
              Please see: https://github.com/nod-ai/SHARK/blob/main/tank/pytorch/torch_model_list.csv""",
     )
-    parser.add_argument(
```
Comment (powderluv) [inline generate_sharktank.py:197]: I guess this is not to commit

**Review by monorimet — APPROVED** [review]
Looks good as long as checks are green

File: tank/model_utils.py [hunk 2]
```diff
@@ -57,8 +59,81 @@ def get_hf_model(name):
     # TODO: Currently the test input is set to (1,128)
     test_input = torch.randint(2, (1, 128))
     actual_out = model(test_input)
+
+    # fx_g = make_fx(
+    #     model(test_input),
```
Comment (pashu123) [inline tank/model_utils.py:64]: You just need to pass the model here.

**Review by monorimet — APPROVED** [review]
LGTM

File: tank/model_utils.py (hunk 2 above)
Comment (monorimet) [reply tank/model_utils.py:64]: Can we resolve and get checks green for now?

Comment (AmosLewis) [reply tank/model_utils.py:64]: Yes

Comment (AmosLewis) [reply tank/model_utils.py:64]: > Can we resolve and get checks green for now?

yes

**Review by pashu123 — APPROVED** [review]
LGTM!

**Pull request merged**